Sources/HummingbirdCore/Request/RequestBody.swift

```diff
@@ -162,13 +162,13 @@ extension RequestBody {
     }
 
     /// A source used for driving a ``RequestBody`` stream.
-    public final class Source {
+    public struct Source: Sendable {
```

Comment:
The change from `class` -> `struct` is the breaking bit right? Can we keep this a class (for now)? It can still be Sendable, as all members are constants and Sendable

Reply:
yeah I've got the Source as a final class in the other PR so we can do that

Reply:
I reverted it to a final class and the benchmarks shows up the fact we have gained an extra allocation. This is a fairly rare path to take so isn't a huge issue. It is just one that the streaming benchmark hits all the time as we have to create streamed request.